pkg/placement/monitoring/metrics.go

```diff
@@ -18,13 +18,31 @@ var (
 		"placement/hosts_total",
 		"The total number of active hosts reported to placement service.",
 		stats.UnitDimensionless)
+	activeActorTypesTotal = stats.Int64(
+		"placement/actorTypes_total",

```

Comment:
Can we use lowercased metric name ?
`placement/actor_types_total`

Reply:
done.